table.csv

```diff
@@ -143,6 +143,7 @@ car-index-sorted,               serialization,  0x0400,         draft,     CARv2
 car-multihash-index-sorted,     serialization,  0x0401,         draft,     CARv2 MultihashIndexSorted index format
 transport-bitswap,              transport,      0x0900,         draft,     Bitswap datatransfer
 transport-graphsync-filecoinv1, transport,      0x0910,         draft,     Filecoin graphsync datatransfer
+transport-gateway-trustless,    transport,      0x0920,         draft,     HTTP car datatransfer
```

Comment:
With "gateway" in the name, am i right to assume this is for car transfer over IPFS HTTP gateway? 
If so, would it make sense to include `ipfs` in the name?

```suggestion
transport-ipfs-gateway-trustless,    transport,      0x0920,         draft,     HTTP IPFS Gateway CAR datatransfer
```

Reply:
yes, if we view this namespace as having a very broad audience, making the assumption that "gateway" bears the singular meaning we think it does isn't appropriate
+1

Reply:
Why would you restrict this to CAR?

Reply:
> do you mean versus also including single blocks, or versus also including unverifiable responses?

I meant single blocks

Reply:
Note: I would prefer knowing what this thing actually is before we merge it as it seems like there are a bunch of unknowns as to what this actually means/how it will be used and the point of the shared table is to enable interoperability (if it wasn't then people could just have system-specific enums and call it a day).

If the desire of the crowd here is to merge this PR before there are any definitions I'd like to request that the entry be removed from the table should it arise that this entry turns out to be abandoned or there is no definition of it in X (maybe 3 or as high as 6?) months.

No we haven't held other entries to this bar (especially the really old ones), but it'd be nice if we could do better going forward here and it doesn't seem like a huge ask that more than the people involved in this PR would be able to figure out what is meant by a the code being added to the table.

Reply:
We can change the comment to
`HTTP IPFS Gateway trustless datatransfer`
is there other wording at this stage you would propose / prefer?

I'm happy to re-visit in 3-6 months to
(a) remove if we aren't using it
(b) revising wording to reflect / point to the specification we've ended up with